dpnp/dparray.pyx

```diff
@@ -47,6 +47,47 @@ cimport numpy
 cimport dpnp.dpnp_utils as utils
 
 
+# initially copyed from original
+cdef class _flagsobj:
```

Comment:
I don not understand why we need this. It is better to reuse this from NumPy

Reply:
We need to get own behavior while we have no a full enough support of features like f-contiguous memory layout, strides, views (sharing the memory) etc. In fact at this moment we just return a constant values of flags and it will be easier to modify our own class "step by step". We can replace it with original Numpy class in future, when it will be possible to reuse original functionality. Possibly I am wrong)

Reply:
@Alexander-Makaryev We "return a constant values" because we support one data state only at this time.
It is questionable we will support diffrent states.
Using Numpy Flags is also possible in "one state"- just construct it based on "constant values".
I think this is ambiguous code what is better to handle via "fallback" because it is not related to SYCL